feature/account/server/certificate/src/main/kotlin/app/k9mail/feature/account/server/certificate/ui/ServerCertificateErrorScreen.kt

```diff
@@ -79,7 +85,27 @@ fun ServerCertificateErrorScreen(
                     .verticalScroll(rememberScrollState())
                     .padding(all = MainTheme.spacings.double),
             ) {
-                TextHeadline4(text = "Warning")
+                Row {
+                    val warningIcon = IconsWithBaseline.Filled.warning
+                    val iconSize = MainTheme.sizes.medium
+                    val iconScalingFactor = iconSize / warningIcon.image.defaultHeight
+                    val iconBaseline = warningIcon.baseline * iconScalingFactor
+
+                    Icon(
```

Comment:
This could be an IconWithBaseline that does the calculation internally.

Reply:
Agreed. But to discuss the approach it felt easier to leave everything in one place.